.yarn/releases/yarn-1.21.0.js

```diff
@@ -6476,7 +6476,7 @@ class Install {
 
             let pattern = name;
             if (!_this.lockfile.getLocked(pattern)) {
-              // when we use --save we save the dependency to the lockfile with just the name rather than the
```

Comment:
This file seems to have got caught in the find and replace. Could you remove it from the PR please?

Reply:
sure, sorry about the delay